src/protocol/damage.rs

```diff
@@ -39,71 +39,81 @@ pub const X11_XML_VERSION: (u32, u32) = (1, 1);
 pub type Damage = u32;
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-#[repr(u8)]
-#[non_exhaustive]
-pub enum ReportLevel {
-    RawRectangles = 0,
-    DeltaRectangles = 1,
-    BoundingBox = 2,
-    NonEmpty = 3,
+pub struct ReportLevel(u8);
+impl ReportLevel {
+    pub const RAW_RECTANGLES: Self = Self(0);
+    pub const DELTA_RECTANGLES: Self = Self(1);
+    pub const BOUNDING_BOX: Self = Self(2);
+    pub const NON_EMPTY: Self = Self(3);
+}
+impl From<ReportLevel> for Option<bool> {
```

Comment:
What would this impl be useful for? I feel like it should just be dropped.

Reply:
As I said above, I'll try to avoid useless `bool` conversions.